install.ps1

```diff
@@ -877,18 +726,203 @@ if (-not $noGui.IsPresent) {
         $newXmlNode.SetAttribute("name", "RAW_TOOLS_DIR")
         $newXmlNode.SetAttribute("value", $rawToolsDirText.text)
 
-        # Add selected packages
-        $packages = $configXml.SelectSingleNode('//packages')
-        foreach($package in $selectedPackagesBox.Items) {
-            $newXmlNode = $packages.AppendChild($configXml.CreateElement("package"))
-            $newXmlNode.SetAttribute("name", $package)
-        }
+
+        [void]$formEnv.Close()
+
     } else {
         Write-Host "[+] Cancel pressed, stopping installation..."
         Start-Sleep 3
         exit 1
     }
 
+    ################################################################################
+    ## PACKAGE SELECTION BY CATEGORY
+    ################################################################################
+
+    # Function that adds the selected packages to the congif.xml for the installation
+    function Install-Selected-Packages
+    {
+      $selectedPackages  = @()
+      foreach ($checkBox in $checkboxesPackages){
+        if ($checkBox.Checked){
+            $package =$checkbox.Text.split(" :")[0]
+            Write-Host ("{0} has been selected" -f $package)
+            $selectedPackages+=$package
+        }
+      }
+      # Add selected packages
+      $packages = $configXml.SelectSingleNode('//packages')
+      foreach($package in $selectedPackages) {
+           Write-Host ($package)
+           $newXmlNode = $packages.AppendChild($configXml.CreateElement("package"))
+           $newXmlNode.SetAttribute("name", $package)
+       }
+    }
+
+    # Function that reset the checkboxes to match the config.xml
+    function Set-InitialPackages {
+        foreach ($checkBox in $checkboxesPackages){
+            $package =$checkbox.Text.split(" :")[0]
+            if (($checkbox.Checked) -and ($package -notin $packagesToInstall)){
+                $checkBox.Checked = $false
+            }else{
+              if ((-not $checkbox.Checked ) -and ($package -in $packagesToInstall)){
+                 $checkBox.Checked = $true
+              }
+            }
+        }
+    }
+    # Function that checks all the checkboxes
+    function Select-AllPackages {
+        foreach ($checkBox in $checkboxesPackages){
+            $checkBox.Checked = $true
+        }
+    }
+
+    # Function that unchecks all the checkboxes
+    function Clear-AllPackages {
+	foreach ($checkBox in $checkboxesPackages){
+            $checkBox.Checked = $false
+        }
+    }
+
+    # Funtion that returns an array of packages that belong to a specific category
+    function Get-PackagesByCategory{
+        param (
+         [string]$category
+        )
+        return $packagesByCategory[$category]
+    }
+
+    Add-Type -AssemblyName System.Windows.Forms
+    [System.Windows.Forms.Application]::EnableVisualStyles()
+
+    $FormCategories                            = New-Object system.Windows.Forms.Form
```

Comment:
[nitpick] Let's stick with came-case with the first letter lowercase, like `$formCategories`